test/test_actions.py

```diff
@@ -125,7 +125,7 @@ def tearDown(cls):
 
         :return: None
         """
-        for resource in ['host', 'service', 'command', 'history',
+        for resource in ['host/srv001', 'service', 'command', 'history',
```

Comment:
I do confirm that it really deletes the host (and only this one) in the backend :wink: